ADR/ADR013-use-paper-trail-gem-for-auditing-and-record-backups.md

```diff
@@ -0,0 +1,51 @@
+# ADR013: Add PaperTrail gem for auditing and record backup
+
+Date: 2023-02-27
+
+## Status
+
+Accepted
+
+## Context
+
+We need way to audit changes to database records to identify what has changed and who made those changes. This information
+will be surfaced to users later on when we come to versioning. This is still valuable to record now for any support tickets
+that come in and require us to understand the form/pages history.
+
+There is also a secondary need be able to restore individual records rather than having to restore from database backups.
+
+## Decision
+
+Add [PaperTrail](https://github.com/paper-trail-gem/paper_trail) as part of draft/live work so we can start recording 
+changes to Forms/Pages but also from any other tables we might want to monitor.
+
+## Consequences
+
+We will not be able to identify what has changed and when without having to go back to splunk server logs and trawl 
```

Comment:
This should be written form the perspective of the consequences of accepting this ADR (this reads as from the perspective or not accepting it and relying on app logging instead).